interface/modules/zend_modules/module/Multipledb/view/multipledb/multipledb/index.phtml

```diff
@@ -0,0 +1,84 @@
+<script>
+    /* +-----------------------------------------------------------------------------+
+     * Copyright 2016 matrix israel
+     * LICENSE: This program is free software; you can redistribute it and/or
+     * modify it under the terms of the GNU General Public License
+     * as published by the Free Software Foundation; either version 3
+     * of the License, or (at your option) any later version.
+     * This program is distributed in the hope that it will be useful,
+     * but WITHOUT ANY WARRANTY; without even the implied warranty of
+     * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+     * GNU General Public License for more details.
+     * You should have received a copy of the GNU General Public License
+     * along with this program. If not, see
+     * http://www.gnu.org/licenses/licenses.html#GPL
+     *    @author  Oshri Rozmarin <oshri.rozmarin@gmail.com>
+     * +------------------------------------------------------------------------------+
+     *
+     */
+    <?php
+    if ($closeBeforeOpening):
+    ?>
+    opener.document.forms[0].submit();
+    window.close();
+
+    <?php
+    endif;
+    ?>
+</script>
+<a href="multipledb/edit"><input type="submit" value="<?php echo  $translate->z_xlt('Add new database') ?>"></a>
+<a href="multipledb/generatesafekey"><input type="submit" value="<?php echo  $translate->z_xlt('Generate safe key') ?>"></a>
+<table class="responstable">
+    <?php
+    $html = null;
+    foreach ($getmultipledb as $value) {
+        $date       = date('Y/m/d H:i:s', strtotime($value['date']));
+        $editLang   = $translate->z_xlt('Edit');
+        $removeLang = $translate->z_xlt('Remove');
+        $html .= <<<html
```

Comment:
this seems odd. this will bypass html escaping it appears. can't just use standard html with echo for php parts or am I missing something?